xarray/tests/test_missing.py

```diff
@@ -23,7 +23,13 @@
     requires_dask,
     requires_scipy,
 )
-from xarray.tests.test_cftime_offsets import _CFTIME_CALENDARS
+
+try:
+    import cftime  # type: ignore # noqa
+
+    from xarray.tests.test_cftime_offsets import _CFTIME_CALENDARS
```

Comment:
sorry i didn't think of this earlier but an alternate and maybe cleaner solution might be moving `_CFTIME_CALENDARS` to `tests/__init__.py` and importing that in both `test_cftime_offsets` and `test_missing.py`

Reply:
Done, I think.